ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-ref.yaml

```diff
@@ -3,7 +3,7 @@ ref:
   cli: latest
   commands: redhat-appstudio-rhtap-installer-e2e-test-commands.sh
   credentials:
-  - mount_path: /usr/local/ci-secrets/redhat-appstudio-qe
+  - mount_path: /usr/local/ci-secrets/rhtap
```

Comment:
this was a copy-paste issue, the location of a secret for rhtap jobs should be `rhtap`